src/Faker/Provider/ng_NG/Person.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Faker\Provider\ng_NG;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{firstNameMale}} {{lastName}}'
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{firstNameFemale}} {{lastName}}'
+    );
+
+    /**
+     * An array of typical Nigerian male firstnames.
+     * @link http://www.top-100-baby-names-search.com/baby-names-nigeria.html
```

Comment:
I could not find an open source link for Nigerian names. So I created a little one, with an aim to expand it in the future.